packages/githublight/style/variables.css

```diff
@@ -0,0 +1,440 @@
+  */
+  --jp-shadow-base-lightness: 0;
+  --jp-shadow-umbra-color: rgba(
+      var(--jp-shadow-base-lightness),
+      var(--jp-shadow-base-lightness),
+      var(--jp-shadow-base-lightness),
+      0.2
+  );
+  --jp-shadow-penumbra-color: rgba(
+      var(--jp-shadow-base-lightness),
+      var(--jp-shadow-base-lightness),
+      var(--jp-shadow-base-lightness),
+      0.14
+  );
+  --jp-shadow-ambient-color: rgba(
+      var(--jp-shadow-base-lightness),
+      var(--jp-shadow-base-lightness),
+      var(--jp-shadow-base-lightness),
+      0.12
+  );
+  --jp-elevation-z0: none;
+  --jp-elevation-z1: 0px 2px 1px -1px var(--jp-shadow-umbra-color), 0px 1px 1px 0px var(--jp-shadow-penumbra-color),
+      0px 1px 3px 0px var(--jp-shadow-ambient-color);
+  --jp-elevation-z2: 0px 3px 1px -2px var(--jp-shadow-umbra-color), 0px 2px 2px 0px var(--jp-shadow-penumbra-color),
+      0px 1px 5px 0px var(--jp-shadow-ambient-color);
+  --jp-elevation-z4: 0px 2px 4px -1px var(--jp-shadow-umbra-color), 0px 4px 5px 0px var(--jp-shadow-penumbra-color),
+      0px 1px 10px 0px var(--jp-shadow-ambient-color);
+  --jp-elevation-z6: 0px 3px 5px -1px var(--jp-shadow-umbra-color), 0px 6px 10px 0px var(--jp-shadow-penumbra-color),
+      0px 1px 18px 0px var(--jp-shadow-ambient-color);
+  --jp-elevation-z8: 0px 5px 5px -3px var(--jp-shadow-umbra-color), 0px 8px 10px 1px var(--jp-shadow-penumbra-color),
+      0px 3px 14px 2px var(--jp-shadow-ambient-color);
+  --jp-elevation-z12: 0px 7px 8px -4px var(--jp-shadow-umbra-color), 0px 12px 17px 2px var(--jp-shadow-penumbra-color),
+      0px 5px 22px 4px var(--jp-shadow-ambient-color);
+  --jp-elevation-z16: 0px 8px 10px -5px var(--jp-shadow-umbra-color),
+      0px 16px 24px 2px var(--jp-shadow-penumbra-color), 0px 6px 30px 5px var(--jp-shadow-ambient-color);
+  --jp-elevation-z20: 0px 10px 13px -6px var(--jp-shadow-umbra-color),
+      0px 20px 31px 3px var(--jp-shadow-penumbra-color), 0px 8px 38px 7px var(--jp-shadow-ambient-color);
+  --jp-elevation-z24: 0px 11px 15px -7px var(--jp-shadow-umbra-color),
+      0px 24px 38px 3px var(--jp-shadow-penumbra-color), 0px 9px 46px 8px var(--jp-shadow-ambient-color);
+  /* Borders
+  *
+  * The following variables, specify the visual styling of borders in JupyterLab.
+  */
+  --jp-border-width: 1px;
+  --jp-border-color0: var(--gh-light-gray0);
+  --jp-border-color1: var(--gh-light-gray1);
+  --jp-border-color2: var(--gh-light-gray2);
+  --jp-border-color3: var(--gh-light-gray3);
+  --jp-border-radius: 2px;
+  /* UI Fonts
+  *
+  * The UI font CSS variables are used for the typography all of the JupyterLab
+  * user interface elements that are not directly user generated content.
+  *
+  * The font sizing here is done assuming that the body font size of --jp-ui-font-size1
+  * is applied to a parent element. When children elements, such as headings, are sized
+  * in em all things will be computed relative to that body size.
+  */
```

Comment:
Same here, I think it deserves a comment that we are intentionally replacing the typeface

Is this also a planned change across all themes? - it feels either it should or should not

Reply:
I did this change because I saw it in other examples and it does help the fonts to be easier to read. I'm planning to keep this across all themes.